editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/BoundaryForm.tsx

```diff
@@ -9,10 +11,16 @@ import { SettingsForm } from "../shared/SettingsForm";
 import { FormProps } from ".";
 
 export default function BoundaryForm({ formikConfig, onSuccess }: FormProps) {
+  const suffixRegEx = /\d{1,10}$/;
```

Comment:
This is a good approach 👍 

We might actually want to restrict the domain as well (e.g. we don't want to allow users to make a request to `https://something-dodgy.com/123456`). I'll check with @ianjon3s but I think an an input prefix here is likely the best way forward.

Reply:
Fixed here - [21cfa53](https://github.com/theopensystemslab/planx-new/pull/3378/commits/21cfa5347c5f13f87945626a6a6b7928ca36a2b8)

For now I've not added an prefix - I think just copy/pasting the full URL is likely an easier user interaction than grabbing the entity ID.